app/models/fika.rb

```diff
@@ -6,13 +6,11 @@ class Fika < ApplicationRecord
   private
 
   def self.participants_uniq_matcher
-    participants = Participant.all
-    possible_participant_pairings = participants.to_a.permutation(2).to_a
+    possible_participant_pairings = Participant.all.to_a.permutation(2).to_a
     unique_pairings = []
     possible_participant_pairings.each do |pair|
       unique_pairings.push(pair.sort_by(&:id))
     end
-    unique_pairings.uniq!
-    unique_pairings
+    unique_pairings.uniq
```

Comment:
Is this change due to formating or refactoring?

Reply:
refactoring

Reply:
This is clever and I totally missed it. When calling `.uniq!` on an array, the expression itself returns the new array if changes are made, but `nil` if all the elements of the array are the same. That is why we needed to return the array by itself on a separate line when using `.uniq!`. We fails some of our tests if this instance method returns `nil`, but the problem is solved by using the regular `.uniq`, which will always return a new array with any duplicates filtered out.

Reply:
👍